bigip.tf

```diff
@@ -26,7 +26,7 @@ resource "aws_secretsmanager_secret_version" "bigip-pwd" {
 module "bigip" {
   # source  = "f5devcentral/bigip/aws"
   # version = "0.1.2"
-  source = "github.com/f5devcentral/terraform-aws-bigip?ref=ip-outputs"
+  source = "github.com/f5devcentral/terraform-aws-bigip?ref=multiple-public-ips"
```

Comment:
if this 'multiple-public-ips' suffix is removed, will it use master? Should we be using that instead since the intent is to merge this to master after approval?

Reply:
If we merge that to master we'll be updating the terraform registry module. there are a number of folk that i need to discuss that with before that happens. in the short term we need to pin this way.

Reply:
Ohhhh, sorry. I missed that part. Nothing to see here...move along...